src/libAtomVM/context.c

```diff
@@ -248,6 +248,15 @@ void context_destroy(Context *ctx)
     free(ctx);
 }
 
+static inline term term_from_context(const Context *ctx)
```

Comment:
I would give it another name such as term_pid_or_port_from_context

Reply:
+1. I had a similar thought, but didn’t come up with a better name to suggest. The function name is pretty vague.

Reply:
agreed. Changed.